api/src/services/authorization.ts

```diff
@@ -496,18 +498,33 @@ export class AuthorizationService {
 			}
 		}
 
-		const preset = permission.presets ?? {};
-
-		const payloadWithPresets = merge({}, preset, payload);
-
 		const fieldValidationRules = Object.values(this.schema.collections[collection].fields)
 			.map((field) => field.validation)
 			.filter((v) => v) as Filter[];
 
+		const hasFieldValidationRules = fieldValidationRules && fieldValidationRules.length > 0;
+
+		if (hasFieldValidationRules) {
+			if (permission.validation && Object.keys(permission.validation).length > 0) {
+				permission.validation = { _and: [permission.validation, ...fieldValidationRules] };
+			} else {
+				permission.validation = { _and: fieldValidationRules };
+			}
+		}
```

Comment:
As this block has been shifted up, I think there should be changes to line 548.
When `hasValidationRules` is `false`, the `fieldValidationRules` may have already been added into `permission.validation`. 

I think that we should be checking on both `hasValidationRules` and `hasFieldValidationRules`. We should also eliminate the wrapping of another `_and:[]` since it was already added upstream.

### Line 548
```
permission.validation = hasValidationRules ? { _and: [permission.validation!] } : { _and: [] };
```

Reply:
I have refactored a lot of this code since in the new $CURRENT_ITEM branch #14887 So might just close this as the fixes this PR is doing are also in that branch?

Reply:
@rijkvanzanten @br41nslug Should we close this in favour of https://github.com/directus/directus/pull/14887 ?

Reply:
I think thats the way to go 👍